frontend/src/wizards/WizardPage.css

```diff
@@ -0,0 +1,3 @@
+.wizard-page .pf-c-drawer__panel {
+  transition: none;
+}
```

Comment:
Is this for consistency with temptifly? If so, might it be nicer to add it there instead? Or is it causing a problem?

Reply:
John says it animates even when it should just be open. No PF switch to disable.